src/main/scala/mesosphere/marathon/core/matcher/base/util/OfferOperationFactory.scala

```diff
@@ -81,9 +80,11 @@ class OfferOperationFactory(
           val persistence = Mesos.Resource.DiskInfo.Persistence.newBuilder().setId(vol.id.idString)
           principalOpt.foreach(persistence.setPrincipal)
 
+          val name = vol.persistentVolume.name.getOrElse(vol.mount.mountPath)
```

Comment:
Using `vol.mount.mountPath` as volume name or just throw "reached" like everywhere else?

Reply:
It is different here. I will add a comment elaboration why it is done the way it is done.